sdk/videoanalyzer/video-analyzer-edge/README.md

```diff
@@ -7,7 +7,10 @@ Use the client library for Video Analyzer Edge to:
 -  Simplify interactions with the [Microsoft Azure IoT SDKs](https://github.com/azure/azure-iot-sdks)
 -  Programmatically construct pipeline topologies and live pipelines
 
-[Product documentation][doc_product] | [Direct methods][doc_direct_methods] | [Source code][source]
+Key links:
+- [Product documentation][doc_product]
+- [Direct methods][doc_direct_methods] 
+- [Source code][source]
```

Comment:
@hivyas could you please populate these links and perhaps use our standard ones?

Reply:
never mind, I just found the links below. However, maybe it is still a good idea to use our standard key links list.